### ISSUE TITLE
Mode is shown as "not installed" during update via "update all"

### ISSUE THREAD

**Issue by hkiel** [benfry/processing4#664]
## Description
Mode is shown as "not installed" during update via "update all"

## Steps to Reproduce
1. nstall the old AndroidMode from #640.
2. click "Updates (1)"
3. update the mode from updates tab (mode immediately disapears on update tab)
4. switch (immediately) to Modes tab and see that AndroidMode is reported to be "not installed" instead of "updating"
5. the mode will appear as installed once it really is fully updated.

Fun fact: when you press "install" during the update, your settings (sdk path) will be lost.

## Your Environment
* Processing version: 4.1.3
* Operating System and OS version: macOS aarch64

**Comment by benfry** [comment]
Yeah, still lots of work to be done in the Contributions Manager. Thanks for letting me know.

**Comment by hkiel** [comment]
Note that updating from the modes tab correctly shows the mode in update process as "updating".
